fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -171,316 +155,56 @@ private void init() {
         times.clear();
         threadIds.clear();
         threadIds.add(Thread.currentThread().getName());
-        MDC.put(ID_KEY, logId);
+        MDC.put(LOG_ID_KEY, logId);
     }
 
     /**
      * Adds the durations in milliseconds of all the recorded timed phases to a map.
      *
-     * @return the map containing all the recorded times per phase in milliseconds
+     * @return the map containing all the recorded times per phase in nanoseconds
      */
-    private Map<String, Long> getDurations() {
+    protected Map<String, Long> getDurations() {
         return times.values()
                 .stream()
-                .peek(
-                        phase -> {
-                            if (phase.start != 0) {
-                                LOG.warn(
-                                        "Exporting duration while timer is running. Measurement might be wrong: {}",
-                                        phase.name
-                                );
-                            }
-                        }
-                )
+                .peek(phase -> {
+                    if (phase.start != 0) {
```

Comment:
If we move the `TimedPhase` out into its own class, then we'll need to make sure to use `TimedPhase::isStarted` instead.